### DISCUSSION TITLE
Metadata file formats

### DISCUSSION THREAD

**Discussion by SandraEliz** [open-sdg/open-sdg#1883] [Q&A]
Hello.

If I wanted to implement this load the metadata with csv files and change the labels as in this section
 
![image](https://user-images.githubusercontent.com/117297044/203595159-84c397a4-e23a-4278-8f29-6d2bf01ac3ee.png)

Should a mapping file be generated for each metadata file?
And where are those files uploaded?

Thank you

**Comment by brockfanning** [comment]
@SandraEliz There should only be one mapping file for the whole site. It can be uploaded anywhere, but needs to be referenced from the configuration file. It might be helpful to see another site using that kind of approach. Here's a site using Excel files for metadata:
* Mapping file: https://github.com/sdg-kyrgyzstan/open-sdg-data-starter/blob/develop/metadata-mapping.csv
* Reference in configuration: https://github.com/sdg-kyrgyzstan/open-sdg-data-starter/blob/develop/open_sdg_config_sdmx.yml#L37

**Comment by SandraEliz** [comment]
Can I have the 2 mapping files?
Both the metadata schema.yml and the one that is generated new

**Reply by brockfanning** [reply]
If I understand right, the two files you mention are used for different purposes. The metadata schema file is required, and is used to inform Open SDG of the fields that should be displayed on the platform. By contrast the metadata mapping file is optional, and is used so that each indicator metadata file can use human-readable labels instead of machine-readable field names.

**Comment by SandraEliz** [comment]
Hello.
I have already generated the metadata-mapping.csv file and added the configuration in the config_data.yml file, but the metadata is not displayed.

![image](https://user-images.githubusercontent.com/117297044/204313995-265c2157-5f2d-423a-902d-8f009ff3e1ed.png)


Am I missing some configuration?

I don't have the _prose.yml file, but I do have the metadata_schema.yml and from it I took the machine readable field names.

Thank you

**Reply by sarah-beasley** [reply]
Hi, can you send the metadata_schema.yml file (or a link to it) so we can have a look at the issue further please?

**Comment by SandraEliz** [comment]
[metadata_schema.txt](https://github.com/open-sdg/open-sdg/files/10125201/metadata_schema.txt)
[metadata-mapping.csv](https://github.com/open-sdg/open-sdg/files/10125206/metadata-mapping.csv)

**Comment by SandraEliz** [comment]
Hello.
I can now view the metadata with the metadata-mapping.csv file.

I no longer understood what this file is for.

I thought that if I put "Descripción" as the label in the metadata-mapping.csv file and in the metadata-schema.yml it is like "SDG_SERIES_DESCR", what would be displayed was the metadata-mapping.csv file

But you see the translation of "SDG_SERIES_DESCR".

This is how it is in the metadata csv file
![image](https://user-images.githubusercontent.com/117297044/205178457-7adf3b46-6973-4ddf-b30a-d93aa157ff34.png)

This is how it is in the metadata-mapping.csv file
![image](https://user-images.githubusercontent.com/117297044/205178516-0d84ed81-5773-4ae4-917d-5665aff36ebc.png)

This is how it is in the metadata_schema.yml file
![image](https://user-images.githubusercontent.com/117297044/205178586-3be42cbf-7269-4a9a-b6b4-10a853c54ca2.png)

And it is visualized
![image](https://user-images.githubusercontent.com/117297044/205179018-1a486f4c-2628-4440-a192-d7e806ce841e.png)

Can you advise me on this issue? please

Thank you

**Reply by sarah-beasley** [reply]
Hi, it might be that your formatting isn't quite correct. You could try to rearrange and correct the spacing by trying the below? If you have multiple fields, you will need to change from 'field' to fields' too.

field:
   - name: SDG_SERIES_DESCR
     element: text
     label: Indicator
     translation_key: metadata_fields.SDG_INDICATOR
     scope: national

**Reply by SandraEliz** [reply]
Hello

It works, now my question is:

Should the tag I added be displayed or what is this metadata configuration file for?
For example:
In the metadata-mapping.csv file I put the "Descripción" tag attached to "SDG_SERIES_DESCR", but it doesn't appear on the page as "Descripción". Appears as "Series".

![image](https://user-images.githubusercontent.com/117297044/205337451-00bfa463-45bf-46d1-9bb5-0cd79f1f5669.png)

**Reply by SandraEliz** [reply]
Hello

Reviewing the documentation again, I saw how to change the text as I mentioned before.

Thank you
